lib/koans/19_streams.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Streams do
+  use Koans
+
+  @intro "Streams"
+
+  koan "Streams are lazier than Enum, they only return a struct" do
+    assert Stream.map([1, 2, 3], &(&1 + 1)).__struct__ == ___
+  end
+
+  koan "You have to ask it explicitly to do the operation" do
```

Comment:
I believe the term often used for these operations are "terminal operations". So you might word this like "Streams are executed by terminal operations"